packages/js/package.json

```diff
@@ -63,7 +65,8 @@
     "lodash.isequal": "^4.5.0",
     "merkletreejs": "^0.2.32",
     "mime": "^3.0.0",
-    "node-fetch": "^2.6.7"
+    "node-fetch": "^2.6.7",
+    "typescript-collections": "^1.3.3"
```

Comment:
I don't see this dependency being used. Is that a peer-dependency requirement?

Reply:
This can and will be removed. It is a peer dependancy for the spl compression sdk. Previously, it was not included in that package, but has since been added.